example/res/values/strings.xml

```diff
@@ -20,9 +20,10 @@
     <string name="confirmPayment">Confirm payment details</string>
     <string name="changeDetails">Change payment details</string>
     <string name="customer_payment_data_example">Customer Payment Selection</string>
-    <string name="launch_customer_session">Launch Customer Session</string>
-    <string name="launch_pay_with_google">Launch Pay with Google</string>
-    <string name="launch_payment_session">Launch Payment Session</string>
+    <string name="launch_customer_session">Customer Session</string>
```

Comment:
what's the diff here?

Reply:
Removed the "Launch" part - it's implied